### DISCUSSION TITLE
How to clone pimcore properly?

### DISCUSSION THREAD

**Discussion by idna001** [pimcore/pimcore#17689] [Q&A / Support]
Hello,

I'm often run into problems while cloning my pimcore projects. Mostly I develop locally with a local file system and local database. 
If everything works as excepted, I deploy my pimcore on the server and make a database dump via the MySQL interface. 

At first, I had issues with the views, but by changing some DB Settings this worked. 
I was able to import my dump successfully. 

The frontend works completely fine. In the backend, though, I can't open documents or objects. 
I get this error:
```
⁠ Status: 500 | Internal Server Error
URL: /admin/page/get-data-by-id
Message: An exception occurred while executing a query: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'id' cannot be null ⁠ 
```

I assume I did something wrong while exporting the sql dump. Or is there maybe a doctrine command that fixes this? 

Thanks for every hint :)
idna

**Comment by Wollitoo** [comment]
did you run `php bin/console pimcore:deployment:classes-rebuild -c -d -n` ?

**Reply by idna001** [reply]
Thanks, but unfortunately the command did not change the behavior of my application.
